Bulldozer.CSV/CSVComponent.cs

```diff
@@ -488,804 +836,2680 @@ public static int GetAttributeFieldType( string attributeTypeString )
-        private const int GroupCreatedDate = 2;             /* Date (mm/dd/yyyy) [Optional] */
-        private const int GroupType = 3;                    /* Guid | Int | String [Required] */
-        private const int GroupParentGroupId = 4;           /* String | Int [Optional] */
-        private const int GroupActive = 5;                  /* "YES" | "NO" [Optional] */
-        private const int GroupOrder = 6;                   /* Int [Optional] */
-        private const int GroupCampus = 7;                  /* String [Optional] */
-        private const int GroupAddress = 8;                 /* String [Optional] */
-        private const int GroupAddress2 = 9;                /* String [Optional] */
-        private const int GroupCity = 10;                   /* String [Optional] */
-        private const int GroupState = 11;                  /* String [Optional] */
-        private const int GroupZip = 12;                    /* String [Optional] */
-        private const int GroupCountry = 13;                /* String [Optional] */
-        private const int GroupSecondaryAddress = 14;       /* String [Optional] */
-        private const int GroupSecondaryAddress2 = 15;      /* String [Optional] */
-        private const int GroupSecondaryCity = 16;          /* String [Optional] */
-        private const int GroupSecondaryState = 17;         /* String [Optional] */
-        private const int GroupSecondaryZip = 18;           /* String [Optional] */
-        private const int GroupSecondaryCountry = 19;       /* String [Optional] */
-        private const int GroupNamedLocation = 20;          /* String [Optional] */
-        private const int GroupDayOfWeek = 21;              /* String [Optional] */
-        private const int GroupTime = 22;                   /* String [Optional] */
-        private const int GroupDescription = 23;            /* String [Optional] */
-        private const int GroupCapacity = 24;               /* Int [Optional] */
+                if ( definedValuesToAdd.Count > 0 )
+                {
+                    rockContext.BulkInsert( definedValuesToAdd );
+                    DefinedTypeCache.Clear();
+                }
+            }
 
-        #endregion Group Constants
+            return definedValuesToAdd.Count();
+        }
 
-        #region Group Polygon Constants
+        /// <summary>
+        /// Adds any attribute categories that are in the csv files (person and family attributes)
+        /// </summary>
+        private void AddAttributeCategories()
+        {
+            var csvCategoryNames = new List<string>();
+            if ( this.PersonAttributeCsvList.Count > 0 )
+            {
+                csvCategoryNames = this.PersonAttributeCsvList.Where( a => !string.IsNullOrWhiteSpace( a.Category ) ).Select( a => a.Category ).Distinct().ToList();
+            }
+            if ( this.FamilyAttributeCsvList.Count > 0 )
```

Comment:
Family attribute is not a person attribute. Need to verify if it is grouptype category and move to group attribute section below

Reply:
@nateh777 per the Rock import model, it appears attribute categories for Family attributes are indeed set up as Person entity qualifiers. https://github.com/SparkDevNetwork/Rock/blob/2c2f2c09e387b85b2d3422e69485b58e0379b592/Rock.Slingshot/SlingshotImport/SlingshotImporter.cs#L2172